build_tools/travis/install.sh

```diff
@@ -62,6 +62,11 @@ if [[ "$DISTRIB" == "conda" ]]; then
     conda create -n testenv --yes $TO_INSTALL
     source activate testenv
 
+    # for python 3.4, conda does not have recent pytest packages
+    if [[ "$PYTHON_VERSION" == "3.4" ]]; then
```

Comment:
If it's possible to use another conda channel, that might be better than overwriting a conda install with pip

Reply:
It's not in defaults, and not in conda-forge, and from a quick search on anaconda.org it does not seem they have their own channel